config/systemd/omnect-ui.service

```diff
@@ -0,0 +1,42 @@
+[Unit]
+Description=omnect ui
+After=time-sync.target docker.service omnect-device-service.service
+Before=first-boot-complete.target
+Wants=first-boot-complete.target docker.service omnect-device-service.service
+ConditionFirstBoot=yes
```

Comment:
do we really want this only on first boot?

Reply:
can you provide an example showing why not?
I think the `docker load` should only happen once. Further we use `docker run -d --restart always` to ensure the container is started after a reboot.
In case there are issues with the container, a factory reset would restore initial settings.